todoman/ui.py

```diff
@@ -98,6 +97,14 @@ def __init__(self, todo, lists, formatter):
                         in widgets.ExtendedEdit.HELP)
         )
 
+        def change_current_list(radio_button, new_state, new_list):
+            self.current_list = new_list
+
+        for _list in self.lists:
+            urwid.RadioButton(items, "labels", state='first True',
```

Comment:
Now all of your radio buttons are called `labels` though, it would be better if you use `_list.name`.